src/sensors/LaserScan.js

```diff
@@ -9,22 +9,19 @@
  * @param options - object with following keys:
  *
  *  * ros - the ROSLIB.Ros connection handle
- *  * topic - the marker topic to listen to
+ *  * topic - the marker topic to listen to (default '/scan')
  *  * tfClient - the TF client handle to use
- *  * color - (optional) color of the points (default 0xFFA500)
- *  * texture - (optional) Image url for a texture to use for the points. Defaults to a single white pixel.
- *  * rootObject (optional) - the root object to add this marker to
- *  * size (optional) - size to draw each point (default 0.05)
- *  * max_pts (optional) - number of points to draw (default 100)
```

Comment:
`color`, `size` and `texture` are removed. Could you check if these fields are set and log deprecation warning with a guide to use the newer way?